build.cake

```diff
@@ -405,16 +406,37 @@ Task("Restore")
     RunTool(env.DotNetCommand, "restore OmniSharp.sln", env.WorkingDirectory)
         .ExceptionOnError("Failed to restore projects in OmniSharp.sln.");
 
+    // Restore test assets
```

Comment:
I'm a bit skeptical of that not being a nightmare. :smile: In working with Cake, I've found myself not being a fan -- especially given that the scripting dialect changes on Windows vs. Unix.

Note that we have to use multiple .NET CLIs to build OmniSharp these days to handle project.json assets. Does Cake support that?